### DISCUSSION TITLE
icon of side bar items?

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#1027] [Q&A] [answered]
https://quarto.org/docs/reference/projects/websites.html#nav-items

If we set a item with `icon` in the sidebar, the item is not displayed.
If we use HTML like `text: <i class="bi bi-twitter"></i>`, it will be displayed, but this will be displayed as a string in navbar, which is a problem when displaying saidebar items in navbar with `sidebar:id`.

Is this the intended behavior? (If so, it would be great if the documentation was updated)

**Comment by jjallaire** [comment, marked as answer]
Yes, this is indeed the intended behavior. I've updated the docs accordingly here: https://github.com/quarto-dev/quarto-web/commit/ac4951a305d293a0e5340535a0cf4011a7a0faee
